lib/datadog/appsec/processor.rb

```diff
@@ -63,6 +86,20 @@ def new_context
         Context.new(self)
       end
 
+      def activate_context
+        context = new_context
+        Processor.send(:active_context=, context)
```

Comment:
What if `active_context` is not `nil`?

Should we allow a stack (i.e `[]#push` on activation and `[]#pop` on deactivation) or should we blow up and raise an exception?

Reply:
I think for simplicity know in which there is only a single processor, we could raise an exception